doc/topics/raw_sealed.rst

```diff
@@ -0,0 +1,46 @@
+=========================
+Raw Sealed Box Encryption
+=========================
+
+.. note::
+
+    While these routines are perfectly safe, higher level convenience
+    wrappers are under development to make these routines easier.
```

Comment:
I copied this from other raw_* documentations. Maybe it is unnecessary for the sealed commands.